src/mame/capcom/cps3.cpp

```diff
@@ -2404,15 +2400,11 @@ void cps3_state::copy_from_nvram()
 
 	/*
 	{
-	    FILE *fp;
-	    const char *gamename = machine().system().name;
-	    char filename[256];
-	    sprintf(filename, "%s_bios.dump", gamename);
-
-	    fp=fopen(filename, "w+b");
+	    auto filename = std::string{ machine().system().name } + "_bios.dump";
+	    auto fp = fopen(filename.c_str(), "w+b");
 	    if (fp)
 	    {
-	        fwrite(rom, 0x080000, 1, fp);
+	        fwrite(romdata, 0x080000, 1, fp);
```

Comment:
We could just delete this section? It was added in v0.117 (2007) and never worked as far as I can tell.
`rom` was never a variable.